emily/handler/Cargo.toml

```diff
@@ -32,6 +34,10 @@ utoipa = "4.2.3"
 warp = "0.3"
 warp_lambda.workspace = true
 
+fake = { version = "2.9.2", features = ["derive", "time"], optional = true }
+
 [dev-dependencies]
 mockito = "0.28"
 mockall = "0.10"
+serial_test = "0.5"
```

Comment:
Is this necessary since you we set `--test-threads==1` when running tests?

Reply:
If someone changes that I'd rather they get defined behavior, seems like something one could accidentally turn off not realizing that the integration tests no longer have a guaranteed order.

Reply:
Worst case scenario is that someone changes this and then tests happen to go in the right order for a while and at some point there's a cargo update that makes some optimization that changes the order. I'd rather just make that impossible, and this library does that.

Reply:
I didn't think this library guaranteed the order. In the project README they state:
> Multiple tests with the serial attribute are guaranteed to be executed in serial. Ordering of the tests is not guaranteed however.

---

But I was curious if this is something that we should consider using in the signer codebase. It seems like it does what `--test-threads==1` but only for the tests that were tagged with the macro. Is that right?

Reply:
ugh, crud. I suspect so, but now I don't know. I'm not sure how we'd guarantee an order.

Reply:
Looks like you can't, I'll find a way around this in the next PR, but for now this should do it's job ~

Reply:
Like the other dependency, I'd change this to
```toml
serial_test = { version = "0.5", default-features = false }
```